identity/src/ecdsa.rs

```diff
@@ -70,6 +70,13 @@ impl fmt::Debug for Keypair {
     }
 }
 
+impl From<self::Keypair> for crate::keypair::Keypair {
+    fn from(kp: self::Keypair) -> Self {
+        #[allow(deprecated)]
+        crate::Keypair::Ecdsa(kp)
+    }
+}
+
```

Comment:
Having this in here creates a cyclic dependency between the two modules. I'd rather have the `From` impls next to the enum.